flow-report/src/common.tsx

```diff
@@ -99,7 +99,7 @@ const FlowStepThumbnail: FunctionComponent<{
       return <FlowStepAnimatedThumbnail frames={frames} width={width} height={height} />;
     }
   } else {
-    thumbnail = fullPageScreenshot?.screenshot.data;
+    thumbnail = Util.getFullPageScreenshot(lhr)?.screenshot?.data;
```

Comment:
```suggestion
    thumbnail = Util.getFullPageScreenshot(lhr)?.screenshot.data;
```

`screenshot` should never be optional right?

Reply:
Yup, you're right. I never bothered to check my assumption that ?. doesn't cascade until now. Thanks.